plugins/ida/ida_main.ml

```diff
@@ -166,12 +252,70 @@ let checked ida_path is_headless =
        (ida_path/"plugins"/"plugin_loader_bap.py"))  >>| fun () ->
   ida_path
 
+let run_ko_symbol_mapper_pass ida_futures =
+  Project.register_pass ~autorun:true ~name:"ko_symbol_mapper" (fun proj ->
+      match Project.get proj filename with
+      | Some file when String.is_suffix file ".ko" ->
```

Comment:
Right now, this pass is triggered on files with a ".ko" extension. Open to other suggestions... (maybe a switch?)